docker/r10k/Dockerfile

```diff
@@ -1,29 +1,40 @@
-FROM puppet/puppet-agent-ubuntu:5.5.1
+FROM ubuntu:16.04
 
 ARG vcs_ref
 ARG build_date
-
-ENV R10K_VERSION="2.5.5"
+ARG version="2.5.5"
+ENV R10K_VERSION="$version"
+ENV UBUNTU_CODENAME="xenial"
 
 LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
       org.label-schema.vendor="Puppet" \
       org.label-schema.url="https://github.com/puppetlabs/r10k" \
       org.label-schema.name="r10k" \
       org.label-schema.license="Apache-2.0" \
-      org.label-schema.version=$R10K_VERSION \
+      org.label-schema.version="$R10K_VERSION" \
       org.label-schema.vcs-url="https://github.com/puppetlabs/r10k" \
-      org.label-schema.vcs-ref=$vcs_ref \
-      org.label-schema.build-date=$build_date \
+      org.label-schema.vcs-ref="$vcs_ref" \
+      org.label-schema.build-date="$build_date" \
       org.label-schema.schema-version="1.0" \
       org.label-schema.dockerfile="/Dockerfile"
 
 RUN apt-get update && \
+    apt-get install --no-install-recommends -y wget ca-certificates lsb-release && \
+    wget https://apt.puppetlabs.com/puppet5-release-"$UBUNTU_CODENAME".deb && \
```

Comment:
Any reason not to install the rolling release package?

Reply:
I don't know that r10k will be able to be used without changes for puppet6.

Reply:
*with changes for puppet6?

Reply:
r10k might need changes to be compatible with puppet6.

Reply:
ahhh I see, I parsed that sentence differently 😅

Reply:
2.5 goes into Davis, 2.6 in everything after that, master/3.0 will be in Puppet 6/Johnson, right?

Reply:
Ok, cool. So yeah, definitely want the puppet5 release package here.

Reply:
So none of this has been merged up to 2.6.x or master (hoyt and later), should we do that now, it sounds like there might be some additional work for master?

Reply:
It would probably be good to get this up at least to 2.6.x, we probably won't build the 3.0 containers until puppet6 is released.